plugins/woocommerce-admin/client/products/use-product-helper.ts

```diff
@@ -152,44 +183,57 @@ export function useProductHelper() {
 			return updateProduct( productId, {
 				...product,
 				status,
-			} ).then(
-				( updatedProduct ) => {
-					if ( ! skipNotice ) {
-						const noticeContent =
-							product.status === 'draft' &&
-							updatedProduct.status === 'publish'
-								? __( 'Product published.', 'woocommerce' )
-								: __(
-										'Product successfully updated.',
-										'woocommerce'
-								  );
-						createNotice( 'success', `🎉‎ ${ noticeContent }`, {
-							actions: getNoticePreviewActions(
-								updatedProduct.status,
-								updatedProduct.permalink
-							),
+			} )
+				.then( async ( updatedProduct ) =>
+					updateVariationOrders(
```

Comment:
Is is possible for us to check if the order has changed before calling this?

Reply:
It's possible to know when the order has changed and it was checked [here](https://github.com/woocommerce/woocommerce/pull/36109/files#diff-d2b049e6eb9611ce0f2904d697fe4098a8d87f4680c2a47cad6ac2804e202ed1R144) inside the `updateVariationOrders` so it does not matters if we call it many times if there are no changes the request is not gonna be made. Also after the request happened the form is been reseted so we just need to check that `variationOrders` is `undefined` to know that no changes had happened.

Reply:
I'm okay with leaving this for a follow-up though I think if we have time, I'd like to fix it up this cycle to avoid tech debt as this will also improve user experience.

Do you want to create the issue or would you like me to?

Reply:
The issue -> https://github.com/woocommerce/woocommerce/issues/36150

Reply:
```suggestion
					updateVariationsOrder(
```